src/gpuapi.rs

```diff
@@ -1,5 +1,5 @@
+#![allow(clippy::upper_case_acronyms)]
```

Comment:
I will take an action item to deal with this, I think renaming the trait as "Gpu" ought to work OK.